modules/renter/contractor/update.go

```diff
@@ -90,6 +80,5 @@ func (c *Contractor) ProcessConsensusChange(cc modules.ConsensusChange) {
 	// maintenance.
 	if cc.Synced {
 		go c.threadedContractMaintenance()
-		go c.managedArchiveContracts()
```

Comment:
now that I see this... I wonder why we ever made it this way??? Should definitely have been doing the saving at the same point that we do the modifications.